minithesis.py

```diff
@@ -261,62 +312,70 @@ def __make_choice(self, n, rnd_method):
         return result
 
 
-class Possibility(object):
+class Possibility(Generic[T]):
```

Comment:
Ah, that makes sense. I don't know how often that would be used (if there were users :-), but you're right in principle, so I'll change this.

Reply:
Yeah, minithesis itself is effectively unused, but ports of it will likely want to port the type signatures too, so it's worth getting the details right.

As far as how often it's likely to be used, I think it's mostly useful for getting the types right with `mix_of` and things that are in some sense "logically" mix-of, but presumably there are some other places where it's convenient too. Covariance tends to be one of those nitpicky little details that doesn't matter right up until it's very annoying to not have it.